server/paymentProviders/paypal/payouts.ts

```diff
@@ -50,16 +51,25 @@ export const payExpensesBatch = async (expenses: any[]): Promise<any[]> => {
     items: expenses.map(getExpenseItem),
   };
 
-  const response = await paypal.executePayouts(connectedAccount, requestBody);
-  const updateExpenses = expenses.map(async e => {
-    await e.update({ data: response.batch_header, status: status.PROCESSING });
-    const user = await models.User.findByPk(e.lastEditedById);
-    await e.createActivity(activities.COLLECTIVE_EXPENSE_PROCESSING, user);
-  });
-  return Promise.all(updateExpenses);
+  try {
+    const response = await paypal.executePayouts(connectedAccount, requestBody);
+    const updateExpenses = expenses.map(async e => {
+      await e.update({ data: response.batch_header, status: status.PROCESSING });
+      const user = await models.User.findByPk(e.lastEditedById);
+      await e.createActivity(activities.COLLECTIVE_EXPENSE_PROCESSING, user);
+    });
+    return Promise.all(updateExpenses);
```

Comment:
Is is possible to have only one expense in the batch failing? Or do the entire batch fails if just one fails?

Reply:
That is a really nice question! I was taking for granted that everything fails.
I just tested to make sure that was the behavior and it is, the whole batch fails.